lib/scout_apm/layaway.rb

```diff
@@ -188,7 +188,7 @@ def format_pid(pid)
     end
 
     def timestamp_from_filename(filename)
-      match = filename.match(%r{scout_(.*)_.*\.data})
+      match = filename.match(%r{scout_(\d+)_\d+\.data})
```

Comment:
What do you think of also adding a `\z` to the end of this regex, to make it clear the filename must be at the end of the string?